code/drasil-example/Drasil/GamePhysics/Body.hs

```diff
@@ -77,7 +77,8 @@ mkSRS = RefSec (RefProg intro [TUnits, tsymb tableOfSymbols, TAandA]) :
    IScope scope_of_requirements_intro_p1 scope_of_requirements_intro_p2, 
    IChar (S "rigid body dynamics") (phrase highSchoolCalculus) (EmptyS), 
    IOrgSec organization_of_documents_intro inModel solution_characteristics_specification EmptyS]) :
-   Verbatim general_system_description :
```

Comment:
GamePhysics is the only case study that relies on `assembler` to produce its sections; this `general_system_description` section was also generated using `assembler`. 
@JacquesCarette @szymczdm Is it alright to make this shift to using `GSDSec`? That is, generally, should `assembler` be used to create sections (mentioned in #321) or the section description languages that were added to issue #368?

Reply:
`assembler` was, I believe, an ambitious attempt to do things in a generalized way, but I believe using `GSDSec` is better for consistency.

Reply:
Create an issue to look into it. I don't remember the details on the implementation presently, since that would have an effect on what we do with it.